checkov/terraform/checks/resource/gcp/GoogleComputeExternalIP.py

```diff
@@ -13,7 +13,7 @@ def __init__(self):
         super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
 
     def scan_resource_conf(self, conf) -> CheckResult:
-        if 'source_instance_template' in conf.keys() and 'access_config' not in conf.keys():
+        if 'source_instance_template' in conf.keys() and 'network_config' not in conf.keys():
```

Comment:
```suggestion
        if 'source_instance_template' in conf.keys() and 'network_interface' not in conf.keys():
```
shouldn't be then `network_interface` too?